debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java

```diff
@@ -273,6 +273,14 @@ else if (dataTypeName.equals("SERIAL")) {
         }
     }
 
+    private Integer parseLength(String lengthStr) {
+        Long length = Long.parseLong(lengthStr);
+        if (length > Integer.MAX_VALUE) {
+            length = (long) Integer.MAX_VALUE;
+        }
```

Comment:
So what's the implication of this? In particular, will we truncate values larger than MAX_VALUE? Or, asked differently, what's the impact of that length value? I believe we'd expose an incorrect value in the optionally exposed field schema metadata; anything else?

Reply:
@gunnarmorling It seems only `BINARY` columns will be affected and in this case it will truncate the length or in case of fixed length it would pad zeros to only max int. But I think these are impacts we can live with. In fact I don't see practical to use that large data in Debezium anyway - you'd need really huge amount of memory.

Reply:
@jpechane, we also should emit a WARN message to the logs in case Integer.MAX_VALUE is exceeded.